lib/galaxy/datatypes/interval.py

```diff
@@ -193,7 +193,8 @@ def set_meta(
     def displayable(self, dataset: DatasetProtocol) -> bool:
         try:
             return (
-                not dataset.dataset.purged
+                not dataset.deleted
+                and not dataset.dataset.purged
```

Comment:
Will the `not dataset.deleted` check make the `not dataset.dataset.purged` check redundant? :thinking:

Reply:
If there's no bug and we don't decide to have purged but not deleted mean that a cleanup script deleted the dataset as opposed to a user action ... I could go either way 🤷

Reply:
Ok! I was just curious if there could be potential situations to have it purged but not deleted :+1: